libraries/ui/src/Card.tsx

```diff
@@ -45,17 +46,20 @@ export const Card: React.FC<CardProps> = ({
       {...(isEntireCardClickable && isExternalUrl && ctaUrl && EXTERNAL_LINK_PROPS)}
       className={wrapperClassName}
     >
-      <div className="card__image-container max-w-[323px] max-h-[223px] w-full mb-3">
-        <img
-          className={`card__image w-full max-h-full object-cover rounded-2xl ${imageClassName}`}
-          src={imageSrc}
-          alt={`${title}`}
-        />
+      <div className="card__image-container max-h-[223px] w-full mb-3">
+        {/* If an element (may or may not be an <img>) is given, use that, otherwise use the image at `imageSrc` */}
+        {imageNode || (
+          <img
+            className={`card__image w-full max-h-full object-cover rounded-2xl ${imageClassName}`}
+            src={imageSrc}
+            alt={`${title}`}
+          />
+        )}
       </div>
 
       <div className="card__content flex flex-col gap-6 w-full flex-1 justify-between">
         <div className="card__text">
-          <h2 className="card__title text-2xl font-[650] text-bluedot-darker mb-1">{title}</h2>
+          <h2 className="card__title text-xl font-[650] text-bluedot-darker mb-1">{title}</h2>
```

Comment:
Flagging that I changed this, different cards in the designs had different sizes and I thought this was the best compromise

Reply:
can we align to the standard semantic definitions? ([globals.css](https://github.com/bluedotimpact/bluedot/blob/6db3fbadd2a7e24ac6ea5567880305d2a06f7c2d/apps/website-25/src/globals.css#L11))
```suggestion
          <h3 className="card__title mb-1">{title}</h3>
```

Reply:
👍, I added an h4 class for this to keep the `text-xl` size:
```css
h4 {
  @apply text-bluedot-darker font-sans text-xl font-[650];
}
```
```tsx
<h4 className="card__title mb-1">{title}</h4>
{subtitle && (<p className={`card__subtitle ${subtitleClassName}`}>{subtitle}</p>)}
```